README.md

```diff
@@ -86,12 +87,39 @@ simp_kubernetes only sets up the services listed above, with a few catches:
 
 The minimal set up would look as follows:
 
+##### On all nodes:
 ```puppet
 class { 'simp_kubernetes':
   etcd_peers   => ['kube-master01.domain.net'],
   kube_masters => ['kube-master01.domain.net','kube-master02.domain.net','kube-master03.domain.net']
 }
 ```
+or in hiera:
+```yaml
+---
+simp_kubernetes::is_master: true
+simp_kubernetes::etcd_peers:
+- kube-master01.domain.net
+simp_kubernetes::kube_masters:
+- kube-master01.domain.net
+- kube-master02.domain.net
+- kube-master03.domain.net
+```
+
+
+##### On the master node (only one is supported at this time):
+```puppet
+class { 'simp_kubernetes':
+  etcd_peers   => ['kube-master01.domain.net'],
+  kube_masters => ['kube-master01.domain.net','kube-master02.domain.net','kube-master03.domain.net'],
+  is_master    => true
+}
+```
+or in hiera:
+```yaml
+---
+simp_kubernetes::is_master: true
+```
```

Comment:
this assumes hiera from all nodes?

Reply:
I hoped to clear this up. I don't want to assume how a hierarchy is set